src/app/(routes)/layout.tsx

```diff
@@ -12,7 +13,14 @@ export const viewport: Viewport = {
 }
 
 const Layout = ({ children }: LayoutProps) => {
-  return <LeftNavLayout>{children}</LeftNavLayout>
+  return (
+    <LeftNavLayout>
+      <div className="mx-auto w-full max-w-[1440px] px-20 pb-8">
+        <HeaderLayout />
+        {children}
+      </div>
```

Comment:
너무 좋네요!
다만 이렇게 작성되면 children 컴포넌트에서 `h-full`을 적용했을 때 스크롤이 생기는 문제가 발생할 것 같습니다.

예를 들어 다음과 같이 적용해보면 좋겠네요!
```suggestion
      <div className="mx-auto max-w-[1440px] w-full px-20">
        <HeaderLayout />
        <div className="pt-8">
          {children}
        </div>
      </div>
```

Reply:
아 그러네요. 저 패딩 값은 레이아웃이 아닌 페이지 컴포넌트(혹은 하위 레이아웃 컴포넌트)에서 알아서 처리해주는게 더 적절해보입니다!